db/seeds.rb

```diff
@@ -33,6 +33,9 @@ def create_user(number)
 # Create members
 5.times do |i|
   user = create_user(i)
+  if i == 1
```

Comment:
We can just use `User.first.add_role ...`

Reply:
I don't know why there is this commit actually. Probably some file that I've touched